src/y_doc.rs

```diff
@@ -12,9 +13,11 @@ use crate::y_xml::YXmlFragment;
 use crate::y_xml::YXmlText;
 use pyo3::prelude::*;
 use pyo3::types::PyBytes;
+use pyo3::types::PyDict;
 use pyo3::types::PyTuple;
+use yrs::updates::decoder::Decode;
 use yrs::updates::encoder::Encode;
-use yrs::Doc;
+use yrs::*;
```

Comment:
I changed this import to be too permissive, let's not `use *` in general

Reply:
Fixed